forms/collections/elements.js

```diff
@@ -86,7 +93,7 @@ define(function (require) {
     /**
      * Gets a list of element models that have failed validation
      * @param  {Number} fieldLimit The number of fields to return. Defaults to 0
-     * @return {Object} An object of the form { errors: [], total, 19, length: 4}
+     * @return {Object} An object of the form { errors: [field, field, field, ...], total, 19, length: 4}
```

Comment:
It could be clearer to split this documentation up using `@typedef`:
- http://usejsdoc.org/tags-typedef.html
- http://usejsdoc.org/tags-callback.html
That is, instead of `{Object} long description` define a custom type and say `{CustomObject} short description` instead?